css/03-generic.css

```diff
@@ -78,6 +78,44 @@ input[type="submit"]:hover {
 	background-color: var( --color-brandGrey );
 }
 
+/* Headers */
+ 
+h2 {
+	background: var( --color-brandGrey );
+	color: var( --color-brandBeige );
+	padding: 1rem;
+	position: relative;
+	transform: rotate(-1.5deg);
+}
+
+h2:before, .entry .entry-title:before {
+	content: none;
+}
+
+.entry .entry-content > h2 {
+	margin-left: -1rem;
+	max-width: none;
+}
+
+h3 {
+	background: var( --color-brandBlack );
+	color: var( --color-brandBeige );
+	padding: 1rem;
+	transform: rotate(1deg);
+}
+
+.entry .entry-content > h3 {
+	max-width: 50%;
```

Comment:
Is this okay on small screens with longer headlines?

Reply:
You are right, it cuts off. I added a media query so this only applies on larger screens.